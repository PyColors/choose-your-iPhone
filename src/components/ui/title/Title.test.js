import React from 'react';
test('Render a h1', () => {
    const wrapper = shallow(
        <h1>Hello Jest!</h1>
    );
    expect(wrapper).toMatchSnapshot();
});
