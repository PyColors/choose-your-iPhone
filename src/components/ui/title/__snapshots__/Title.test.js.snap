// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render a h1 1`] = `
<h1>
  Hello Jest!
</h1>
`;
